qemu/tests/timedrift_adjust_time.py

```diff
@@ -89,12 +88,12 @@ def execute(self, cmd, session=None):
             timeout = int(self.params.get("execute_timeout", 360))
             ret = session.cmd_output(cmd, timeout=timeout)
         else:
-            ret = utils.system_output(cmd)
+            ret = process.system_output(cmd)
```

Comment:
Hi, in my previous test, the `utils.system_output()` can not be directly replaced by `process.system_output()` since `utils.system_output()` can run command no matter the command is a builtin command or not. `process.system_output(cmd, shell=True)` can have the same function.

Reply:
Is this the only place I need to change?

Reply:
Same issue with `process.run()` and `process.system()` I think.